pkg/gardenlet/operation/botanist/vpnshoot.go

```diff
@@ -40,6 +41,12 @@ func (b *Botanist) DefaultVPNShoot() (component.DeployWaiter, error) {
 		SeedPodNetwork:                       b.Seed.GetInfo().Spec.Networks.Pods,
 	}
 
+	if !gardencorev1beta1.IsIPv6SingleStack(b.Shoot.GetInfo().Spec.Networking.IPFamilies) {
+		values.ShootPodNetwork = *b.Shoot.GetInfo().Spec.Networking.Pods
+		values.ShootServiceNetwork = *b.Shoot.GetInfo().Spec.Networking.Services
+		values.ShootNodeNetwork = *b.Shoot.GetInfo().Spec.Networking.Nodes
```

Comment:
Please rather use `b.Shoot.Networks` instead.

Reply:
that's going to be fun because `b.Shoot.Networks` is an array of `net.IPNet` elements with both IPv4 and IPv6 families combined but we only need one IPv4 network as a string here. I'll add some helper functions...